opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -623,6 +625,7 @@ def __init__(
         instrumentation_info: InstrumentationInfo = None,
         record_exception: bool = True,
         set_status_on_exception: bool = True,
+        limits=_UnsetLimits,
```

Comment:
is this change intentional? do we want to have unset as default?

Reply:
Looks like `limits` is always passed down from tracer so this is fine I guess.

Reply:
> is this change intentional? do we want to have unset as default?

From a span's perspective, yes, I think it should not deal with any default values logic. provider does that already.